pandas/io/parsers.py

```diff
@@ -6,10 +6,11 @@
 import csv
 import datetime
 from io import BufferedIOBase, RawIOBase, StringIO, TextIOWrapper
+from itertools import chain
```

Comment:
My preference would be to:
```suggestion
import itertools
```
and in the code use `itertools.chain`. That's easier to read in my opinion, as it makes clear that chain is not a variable name or a custom function, but the itertools function.